llvm/lib/CheerpUtils/Utility.cpp

```diff
@@ -1045,6 +1045,8 @@ std::vector<Constant*> getGlobalConstructors(Module& module)
 
 bool needsSecondaryName(const Value* V, const PointerAnalyzer& PA)
 {
+	if (isa<InvokeInst>(V))
+		return true;
```

Comment:
Why?

Reply:
I have a testcase, will find it, where a second name was needed for an Invoke but was not otherwise tracked. Unsure whether the problem was somehow related to Invokes in particular. Will find the proper test-case, I can remove it for now.

Reply:
Removed from this PR.